vql/parsers/plist.go

```diff
@@ -57,14 +79,86 @@ func (self *PlistParser) Call(ctx context.Context,
 	return val
 }
 
-func (self PlistParser) Info(scope *vfilter.Scope, type_map *vfilter.TypeMap) *vfilter.FunctionInfo {
+func (self _PlistPlugin) Call(
+	ctx context.Context,
+	scope *vfilter.Scope,
+	args *ordereddict.Dict) <-chan vfilter.Row {
+	output_chan := make(chan vfilter.Row)
+
+	go func() {
+		defer close(output_chan)
+
+		arg := &_PlistPluginArgs{}
+		err := vfilter.ExtractArgs(scope, args, arg)
+		if err != nil {
+			scope.Log("plist: %s", err.Error())
+			return
+		}
+
+		
+		err = vql_subsystem.CheckFilesystemAccess(scope, arg.Accessor)
+		if err != nil {
+			scope.Log("plist: %s", err)
+			return
+		}
+
+		for _, filename := range arg.Filenames {
+			func() {
+				defer utils.RecoverVQL(scope)
+
+				accessor, err := glob.GetAccessor(arg.Accessor, scope)
+				if err != nil {
+					scope.Log("plist: %v", err)
+					return
+				}
+
+				file, err := accessor.Open(filename)
+				if err != nil {
+					scope.Log("Unable to open file %s: %v",
+						filename, err)
+					return
+				}
+				
+				defer file.Close()
+
```

Comment:
Typically Go projects run the code through gofmt to get the spacing and formatting right.

Reply:
Okay, I've used `gofmt` to format `plist.go` and added another commit.